doc/sphinx/cython/lxcat_conversion.rst

```diff
@@ -0,0 +1,10 @@
+**************************
+LXCat to YAML conversion
+**************************
+
+For documentation of the command line interface, see :ref:`sec-lxcat2yaml`.
```

Comment:
Just remove this line - there is no other documentation available at the moment (you'd have to write this).